packages/ui5-middleware-simpleproxy/lib/proxy.js

```diff
@@ -82,6 +101,9 @@ module.exports = function ({ resources, options }) {
       if (providedStrictSSL === false) {
         proxyReqOpts.rejectUnauthorized = false;
       }
+      if (providedHttpHeaders) {
+        proxyReqOpts.headers = Object.assign(proxyReqOpts.headers, providedHttpHeaders); 
```

Comment:
In case of using `Object.assign` the first parameter is the target which will be extended with the values from the additional parameters. In this case here it doesn't make a difference as you assign the return value of the `Object.assign` to the first parameter. Just FYI if you want to make a copy the first parameter of `Object.assign` should be an empty object.

Reply:
I know you said I don't need to do changes, but I removed this returning 😁